ui/src/pages/faq.tsx

```diff
@@ -104,7 +104,7 @@ const faqContents = [
         id: 'publish_octopus',
         heading: 'Can I publish in both Octopus and a journal?',
         content:
-            "<p className='mb-2'>Octopus is designed to sit alongside journals and other dissemination outlets, allowing these channels to specialise in delivering key findings to their audiences while Octopus acts like a ‘patent office’ to record who has done what and when, and ensure the quality, integrity and accessibility of all primary research, in full.</p><p className='mb-2'>Recording work to Octopus should be no barrier to publishing a traditional paper. However please note that journal policies differ in how they treat material that has been previously published, such as on a preprint server. If in doubt, we recommend that you check with your intended journal before publishing in Octopus.</p>"
+            "<p className='mb-2'>Octopus is designed to sit alongside journals and other dissemination outlets, allowing these channels to specialise in delivering key findings to their audiences while Octopus acts like a ‘patent office’ to record who has done what and when, and ensure the quality, integrity and accessibility of all primary research, in full.</p><p className='mb-2'>Recording work to Octopus should be no barrier to publishing a traditional paper. However please note that journal policies differ in how they treat material that has been previously published, such as on a preprint server. If in doubt, we recommend that you check with your intended journal before publishing in Octopus.</p><p classname='mb-2'><em>Royal Society Open Science</em> will consider for publication manuscripts based on work that has been previously recorded in Octopus. To learn more about publishing with Royal Society Open Science, please visit <a href='https://royalsocietypublishing.org/rsos/for-authors' class='underline'>https://royalsocietypublishing.org/rsos/for-authors</a>.</p>"
```

Comment:
i think the link **https://royalsocietypublishing.org/rsos/for-authors** should open in a new tab since being external

Reply:
That's true. Thanks